tools/FakeItEasy.Build/Program.cs

```diff
@@ -56,6 +56,19 @@
     forEach: projectsToPack,
     action: project => Run("dotnet", $"pack {project.Path} --configuration Release --no-build --nologo --output \"{Path.GetFullPath("artifacts/output")}\""));
 
+// If mkdocs's site_url configuration is not set, sitemap.xml is full of "None" links.
+// Even with a valid site_url, the link check would fail if we added a new page.
+// So trust that sitemap.xml will be generated correctly.
+// The 404.html page is likewise generated, and isn't even the one we use on the live site (mike generates a new one).

```

Comment:
I'm confused by this comment. If I understand correctly, because site_url is not set, the sitemap.xml is not generated correctly. But then you say that we "trust that sitemap.xml is generated correctly"?